components/profile/Tab_list.tsx

```diff
@@ -16,12 +16,10 @@ const Tab_list=(props)=>{
 	  return(
 	  	<>
 		<Tabs tabPosition={props.position} size={'large'} animated={false} onTabClick={key=>props.onClick(key)}>
-		     <TabPane tab="Posts" key="Posts">
-		     </TabPane>
-		     <TabPane tab="Followers" key="Followers">
-		     </TabPane>
-		     <TabPane tab="Following" key="Following">
-		     </TabPane>
+			{[...props.tabs].map(i=>(
```

Comment:
Hey Ayush, I realized that it would be better if we implemented the tab system with ant design's tab, https://ant.design/components/tabs/. Since you made the styles for it already, can you restyle it and use their logic instead? I think we will be more bug free if we do this.

Reply:
Already styled